public_html/common.php

```diff
@@ -67,8 +69,10 @@ function load_wiki_name($query_array) {
 	  'wiktionary' => 'wiktionary',
 	);
 	$short_project = $project_map[$project];
-	if ( $project === 'wikidata' ) {
-		$wiki_name = 'wikidatawiki';
+
+	// We set database name directly if received as a valid parameter
+	if ( !(empty($database)?) ) {
```

Comment:
This does not work, although I'm not sure why. I put this version up to test, and it returns no results unless I change this line.